dev/tools/dartdoc.dart

```diff
@@ -109,7 +109,10 @@ dependencies:
     '--header', 'analytics.html',
     '--header', 'survey.html',
     '--footer-text', 'lib/footer.html',
-    '--exclude', 'temp_doc',
+    '--exclude-packages',
+'analyzer,args,barback,cli_util,csslib,front_end,glob,html,http_multi_server,io,isolate,js,kernel,logging,mime,mockito,node_preamble,plugin,shelf,shelf_packages_handler,shelf_static,shelf_web_socket,utf,watcher,yaml',
```

Comment:
i'm surprised to see `io` and `plugin` in this list, but I'm not sure exactly which those refer to... is it dart:io and something to do with flutter plugins, or something else?

Reply:
These refer to: https://pub.dartlang.org/packages/io and https://pub.dartlang.org/packages/plugin, both included in the transitive package dependencies.  

The exclude-packages line was constructed to minimize the change with the new recursive package handling.  We can change it; the current value was constructed mechanically.